src/Controls/tests/UITests/Tests/Issues/Issue15815.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue15815 : _IssuesUITest
+{
+	public Issue15815(TestDevice device)
+		: base(device)
+	{ }
+
+	public override string Issue => "Horizontal CollectionView does not show the last element under some condition";
+
+	[Test]
+	public void LastItemIsVisilbe()
+	{
+		var lastItem = App.WaitForElement("id-2");
+		Assert.AreEqual("Item 2", lastItem.GetText());
```

Comment:
Is this test failing before this PR? I think I saw from the issues that the item is there, just not scrollable. Should we add a visual test that confirm the collection can actually scroll it into view?

Reply:
Yes failing , but we can add more tests late https://github.com/dotnet/maui/issues/20654